### ISSUE TITLE
Check for valid show folder when double clicking an .xsq

### ISSUE THREAD

**Issue by derwin12** [xLightsSequencer/xLights#4997]
If a user double clicks and .xsq (not the new .xsqz), it will open xlights and point to a new temporary show folder if needed where the sequence was found. Instead of doing that .. show that warning that is coded that says "this doesnt look like a valid show folder". Either dont change the showfolder (use their permanent) or just exit out??

**Comment by cybercop23** [comment]
I tend to agree. Didn't know this was here. We should STOP users from being able to double click on a sequence.

**Comment by computergeek1507** [comment]
I tend to disagree it's working as intended
